lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -183,8 +195,74 @@ class RenderForm extends React.Component {
     );
   }
 
+  handleInputChange(event) {
+    event.preventDefault();
+    const queryString = event.target.value;
+    if (queryString.length > 3) {
+      if (this.typingTimeout) { clearTimeout(this.typingTimeout); }
+      const getSuggestions = async () => {
+        const response = await fetch(`${this.zendeskApiUrl}/api/v2/help_center/articles/search.json?query=${queryString}`, {
+          headers: {
+            'Content-Type': 'application/json',
+          },
+        });
+        let suggestions = await response.json();
+        suggestions = suggestions.results.slice(0,6);
+        this.setState({ suggestions });
+      };
+      this.typingTimeout = setTimeout(async () => {
```

Comment:
Isn't typingTimeout react state? Shouldn't you be using setState to update it?

Reply:
Changing this to setState here.